utils/testsuite.c

```diff
@@ -140,9 +141,18 @@ const char *corrupt_types[] = {
     "CORRUPT_SIZE"
 };
 
-void fatal(char *str)
+__PRINTFLIKE(1)
+void fatal(const char *str, ...)
 {
-    printf("Failed with: %s\n",str);
+    va_list ap;
+
+    fflush(stdout);
```

Comment:
There's no need for this fflush, why are you using it?

Reply:
True. Noneed for it.

Reply:
Sorry, been busy - thanks @bgermann for taking this on.

The '\n' in the original printf(), on line 145, one would think should have flushed the output.  However in the testing I was doing on Solaris and Oracle Linux 9 it was missing previously written detail which adding the fflush() then showed.  Alas I can not see any negative results in my notes, only those with the additional debug provided with the changes.  Leave it as it is, when I get around to testing again I can open a new issue if necessary.